### PR TITLE
Feat/83 invoke originate master

### PR DESCRIPTION
**Pull request by Sm00g15**

### CODE REVIEW

File: src/tezos/TezosTypes.ts [hunk 1]
```diff
@@ -82,5 +82,43 @@ export interface Operation {
     delegatable?: boolean,
     destination?: string,
     amount?: string,
-    script?: string
+    script?: string | object;
 }
+
+export interface ContractOriginationOperation {
```
Comment (anonymoussprocket) [inline src/tezos/TezosTypes.ts]: Why not `extends` Operation?

File: integration_test/TezosConseilQuery.spec.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('Block fetchers', () => {
         const head = await TezosConseilQuery.getBlockHead(conseilURL, conseilApiKey);
         expect(head.hash.startsWith('B')).to.equal(true);
         const aBlock = await TezosConseilQuery.getBlock(conseilURL, head.hash, conseilApiKey);
-        expect(aBlock.block.hash).to.equal(head.hash);
+        expect(aBlock["'block"].hash).to.equal(head.hash);
```
[the remark this answers is not in the document]
Comment (Sm00g15) [reply integration_test/TezosConseilQuery.spec.ts]: removed line returns undefined, added line returns the block hash

File: integration_test/TezosConseilQuery.spec.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import 'mocha';
 import {expect} from 'chai';
 import {TezosConseilQuery} from '../src'
-import {servers} from "./servers";
+import {servers} from "../test/servers";
```
[the remark this answers is not in the document]
Comment (Sm00g15) [reply integration_test/TezosConseilQuery.spec.ts]: on which branch? doesn't seem to be in master : https://github.com/Cryptonomic/ConseilJS/tree/master/integration_test

File: src/tezos/TezosTypes.ts (hunk 1 above)
Comment (Sm00g15) [reply src/tezos/TezosTypes.ts]: good call...implemented

**Pull request merged**